### DISCUSSION TITLE
How to find depednecies to install

### DISCUSSION THREAD

**Discussion by xiaoxiang75** [brianhie/viral-mutation#36] [Q&A]
Want to run the program, but not sure where can find depdencies to install ??

**Comment by brianhie** [comment]
Here are some requirements: https://github.com/brianhie/viral-mutation/blob/master/requirements.txt

**Reply by xiaoxiang75** [reply]
Thank you very much- Try to find each of them, do you have any suggestions where I can get to run ?

**Reply by brianhie** [reply]
You can install these with `pip install -r requirements.txt`
